<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Drive Controller</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(x1:1, y1:1, x2:1, y2:0, stop:0 rgb(30, 30, 30), stop: 0.8 rgb(30, 30, 30), stop:1 rgb(16, 150, 230));
	font-family: &quot;Calibri&quot;;
	font-size: 16pt;
	color: white;        
}

QLabel {
	background-color: rgba(0, 0, 0, 0);
}

QLabel#title_label {
	font-family: &quot;Impact&quot;;
	font-size: 30pt;
}

QPushButton {
	padding: 20px;
	margin: 10px; 
	font-family: &quot;Arial Black&quot;;
	font-weight: bold;
}

QPushButton#connect_button {
	background-color: green;
}

QPushButton#connect_button:hover {
	background-color: rgb(66, 202, 66);
}

QPushButton#settings_button {
	background-color: rgb(64, 64, 64);
}

QPushButton#settings_button:hover {
	background-color: rgb(100, 100, 100);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>Remote Drive Controller</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Для подключения контроллера необходимо установить на телефон приложение контроллера. После этого в приложении нужно указать IP текущего ПК, а также порт, который должен совпадать с портом подключения в настройках. Для работы аналогового управления неободима установка драйвера VJoy, который можно скачать &lt;a href=&quot;https://github.com/shauleiz/vJoy/releases&quot;&gt;здесь&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>80</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Статус:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="connect_status_label">
        <property name="text">
         <string>Контроллер не подключен</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>Подключиться</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settings_button">
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
